tox.ini

```diff
@@ -13,120 +13,24 @@ skipsdist = true
 setenv =
     QT_QPA_PLATFORM_PLUGIN_PATH={envdir}/Lib/site-packages/PyQt5/plugins/platforms
     PYTEST_QT_API=pyqt5
+    pyqt{,56,571,58,59}: QUTE_BDD_WEBENGINE=true
+    cov: PYTEST_ADDOPTS=--cov --cov-report xml --cov-report=html --cov-report=
 passenv = PYTHON DISPLAY XAUTHORITY HOME USERNAME USER CI TRAVIS XDG_* QUTE_* DOCKER
+basepython =
+    py35: python3.5
+    py36: python3.6
 deps =
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/misc/requirements/requirements-tests.txt
+    pyqt: PyQt5
```

Comment:
Please use `-r{toxinidir}/misc/requirements/requirements-pyqt.txt` instead - that should always be the latest version, but also makes sure we don't have random breakage because of updates.